homework_4/css/style.css

```diff
@@ -0,0 +1,179 @@
+body {
+	background-image: url(../img/background.jpg);
+}
+
+.box {
+	width: 1000px;
+	height: 998px;
```

Comment:
your container must be flexible by height and depends on count of content inside